packages/web-twig/src/Resources/components/Tooltip/stories/Tooltip.twig

```diff
@@ -0,0 +1,90 @@
+<section class="docs-Section">
```

Comment:
Can you please move each section to a separate file for better readability and maintainability?